src/awesome_map/BoundaryInterceptor.js

```diff
@@ -252,6 +350,9 @@ define(function(require) {
 
             case EventTypes.MOUSE_WHEEL:
 
+                this._updateBoundariesStillVisible();
+                this._debouncedDetermineBoundaryVisibility();
```

Comment:
And this, _debouncedDetermineBoundaryVisibility basically sets a new this._visibleBoundaries.  So, the call to this._updateBoundariesStillVisible(); seems like it is useless here.  (And, the order is different than drag end.)